src/v2/Components/ArtworkFilter/Utils/urlBuilder.tsx

```diff
@@ -46,18 +56,24 @@ export const buildUrl = (state: ArtworkFilters, pathname?: string): string => {
   return url
 }
 
-export const updateUrl = (state: ArtworkFilters) => {
-  const url = buildUrl(state)
+export const updateUrl = (
+  state: ArtworkFilters,
+  defaultValues?: Partial<ArtworkFilters>
+) => {
+  const url = buildUrl(state, { defaultValues })
 
   if (typeof window !== "undefined") {
     window.history.replaceState({}, "", url)
   }
 }
 
-export const removeDefaultValues = (state: ArtworkFilters) => {
+export const removeDefaultValues = (
+  state: ArtworkFilters,
+  defaultValues?: Partial<ArtworkFilters>
+) => {
```

Comment:
So that the function signatures are all the same across the board, you can share the newly created interface in both `updateUrl` and `removeDefaultValues`: 

```
state: ArtworkFilters,
options: BuildUrlOptions
```

and now all of the code (even how one accesses `defaultValues` is consistent). Since the second `options` param is an object, it's fine that the latter two functions don't actually use `pathname`.